### DISCUSSION TITLE
Multiple port

### DISCUSSION THREAD

**Discussion by hnothing2016** [bcakmakoglu/vue-flow#1533] [Q&A]
![微信截图_20240709194640](https://github.com/bcakmakoglu/vue-flow/assets/20417103/1b97676e-5aaf-47ce-9303-7b8dc3a9fcb7)
support multiple port ，like node editor

**Comment by bcakmakoglu** [comment]
Thanks for the post *but*
1. If you want to request a feature, please write a proper feature request
2. Check if the library already supports that (it does, use a custom node with as many handles as you want.). The Math example has a node like that already so please check examples first before opening a thread: https://vueflow.dev/examples/math.html
3. Check if there are already existing feature requests for the same thing.
